arroyo-df/src/lib.rs

```diff
@@ -158,12 +156,14 @@ impl ArroyoSchemaProvider {
                 #[allow(deprecated)]
                 ScalarUDF::new(
                     "unnest",
-                    &Signature::any(1, Volatility::Immutable),
+                    &Signature::any(1, Volatility::Volatile),
```

Comment:
I think this is volatile so that DF doesn't replace a constant term. If that's the case, wanna leave a comment to that affect?

Reply:
That's correct, I'll add a comment